sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -153,6 +152,98 @@ object GpuOrcScan {
       meta.willNotWorkOnGpu("mergeSchema and schema evolution is not supported yet")
     }
   }
+
+  private lazy val numericLevels = Map(
```

Comment:
nit: looks like we only care about the relative ordering of levels. So we could just do a 
```scala
private lazy val numericLevels = Seq(
   DType.DTypeEnum.BOOL8, 
   DType.DTypeEnum.INT8,
   ...
 ).zipWithIndex.toMap 
```

Reply:
nice suggestion